.travis.yml

```diff
@@ -28,6 +30,7 @@ matrix:
 
 before_install:
   - set -eo pipefail
+  - curl -O -L https://github.com/oradwell/covers-validator/releases/download/v0.5.0/covers-validator.phar
```

Comment:
I must say I'm not a big fan of that solution for two reasons;

- It requires a download on each build. You could move it in a special directory which is cached to avoid that but then comes the problem of managing that cache: when do you invalidate it i.e. when do you download again that file?
- It is no longer tracked in the project, i.e. a contributor (myself included) no longer knows that this tool is being used before submitting a build.

Both can be acceptable tradeoff to be fair, but I think an easier solution would be to add `oradwell/covers-validator` in a dedicated bamarni namespace (as it is done for symfony and the mutation testing). To be honest I think this is what should have been done in the first place, my bad for not noticing and flagging this in the PR introducing `oradwell/covers-validator`.